demo-shell/src/app/components/files/files.component.ts

```diff
@@ -336,7 +337,7 @@ export class FilesComponent implements OnInit, OnChanges, OnDestroy {
 
         if (this.contentService.hasPermission(contentEntry, 'update')) {
             this.dialog.open(VersionManagerDialogAdapterComponent, {
-                data: { contentEntry:contentEntry, showComments:showComments, allowDownload :allowDownload},
+                data: { contentEntry: contentEntry, showComments: showComments, allowDownload: allowDownload },
```

Comment:
That's the ES6 feature if I remember correctly. Using single variable causes the JSON property to be called the same, so `{ property }` equals `{ property : property }`.

Reply:
not sure is part of the destructuring this